Code/GraphMol/FileParsers/testMolSupplier.cpp

```diff
@@ -103,6 +104,26 @@ int testMolSup() {
     }
     TEST_ASSERT(i == 16);
   }
+  {
```

Comment:
I’d want to see at least one test that validates that the structure read is actually correct (maybe with canonical SMILES).
Also one to verify that coordinates are being in correctly.

Reply:
Added test that validates smiles of first structure